### ISSUE TITLE
accessibility: remove tabindex from DataTable headers when not sortable

### ISSUE THREAD

**Issue by kenirwin** [Miamiohlibs/Argus#125]
This should be fixed by [react-data-table-component](https://github.com/jbetancur/react-data-table-component) 7.7.1
